src/megaapi_impl.cpp

```diff
@@ -170,6 +174,8 @@ MegaNodePrivate::MegaNodePrivate(MegaNode *node)
         this->height = node->getHeight();
         this->shortformat = node->getShortformat();
         this->videocodecid = node->getVideocodecid();
+        this->favourite = node->isFavourite();
+        this->label = static_cast<nodelabel_t>(node->getLabel());
```

Comment:
IIRC, in the ctor you can access to the members directly (avoiding the casting, in this case)
```suggestion
        this->label = node->label;
```

Reply:
MegaNode::getLabel returns int to ensure it's compatible with bindings, and label type is nodelabel_t, so you need a casting to make the assignation